Update theme

johnholdun opened the pull request:
(This builds on #120 and #122 so please review just the last commit. I'll rebase as those are updated and merged. Thanks!)

Adds some enhancements to the theme/UX. There are some DX changes as well, like making sure all the colors in the CSS file use variables. This is mostly subjective, so if there's anything I should alter or roll back, please don't hesitate to ask!

See it live here, and ask me privately for the password if you want to see the admin stuff: https://befitting-innate-roadway.glitch.me

[ed. note: fixes #81 and fixes #4 btw! -ckolderup]

johnholdun commented on src/pages/layouts/main.hbs:
```diff
@@ -34,11 +34,17 @@
         {{{body}}}
       </div>
+      {{#if admin}}
+        {{> admin_subnav}}
```
I merged the main and admin templates into one—they were mostly the same save for the admin subnav showing up conditionally. I mad that condition a parameter of the main theme.

andypiper commented:
Just an observation on the test site - this looks *really* nice, and as a bonus, seems to also resolve the weirdness that causes the buttons to show as white text on white on my iOS browser for the existing default theme. Nice work!

ckolderup commented:
We talked about this a bit on IRC today but yeah this is looking _crisp_ @johnholdun, thank you, and it's a big relief to see things resizing properly on mobile as well. 

If you have a chance to get that admin nav stuff appearing before the admin page content like we discussed I think I'd be very happy to merge this along with the first PR.

ckolderup commented:
if I say fixes #81 in a comment, will it link things properly? I use Jira at work, sob

andypiper commented:
Oh, is there an IRC?

ckolderup commented:
not right now! John and I have just been in a group chat that never left IRC, so it's our primary way of talking to each other.

johnholdun commented:
Added another commit here that changes the order of the admin nav to appear before main content on small screens, _and_  makes the admin links programmatic so that the link to the current page is unlinked and gets a different visual treatment (on all screen sizes). The changes are on the Glitch if you want to see.

I squashed #123 into the first commit here and forced-push so I think that this can be merged after that one without conflicts, but I'll re-rebase if that ends up not being true.

ckolderup commented:
it appears to be not true, sorry. I'll get committer access set up for you & the other current contributors soon and then we can use branches in the repo that we can all collaborate on more easily.

johnholdun commented:
Haha I messed that up so bad. Rebased again though!

ckolderup commented:
thanks! I'm so happy to have a mobile-friendly view now 😊
